### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adf-ncgr**
added some details about CHECKSUM files, basic instructions and rationale

### CODE REVIEW

**Comment by adf-ncgr** [comment]
@StevenCannon-USDA let me know if these changes conform to your understanding of metadata management. @el239 let me know if anything seems unclear

**Review by StevenCannon-USDA — COMMENTED** [review]
Looks good to me, but could be expanded. I suggest the following (more or less; please season to taste):
<hr>

Several types of files should be checked for valid format and contents prior to pushing to the datastore-metadata repository:
```
  validate.sh description_genus_species YAML_FILE
  validate.sh description_genus         YAML_FILE
  validate.sh readme                    YAML_FILE
```


The metadata files in each collection should also be compressed and indexed, using the following script, which is found in the `scripts/` directory of this repository:

```
  compress_and_index.sh COLLECTION
```

Also, calculate md5 sums for the files in the collection with the following script - again, found in the `scripts/` directory:
```
  mdsum-folder.bash COLLECTION
```

For additional description of protocols related to data curation for the data store, see https://github.com/legumeinfo/datastore-specifications/tree/main/PROTOCOLS

**Comment by adf-ncgr** [comment]
Thanks @StevenCannon-USDA I'm happy to make changes but just so I'm clear you are referring to scripts in the datastore-specifications repo, correct? And is the validate.sh script you reference this one (which for some reason isn't under scripts and lacks an extension)?:
https://github.com/legumeinfo/datastore-specifications/blob/main/validate

thanks for the reminder about the detailed PROTOCOLS writeups

**Comment by StevenCannon-USDA** [comment]
> ... you are referring to scripts in the datastore-specifications repo, correct? 

Right. My mistake.

> And is the validate.sh script you reference this one (which for some reason isn't under scripts and lacks an extension)?: https://github.com/legumeinfo/datastore-specifications/blob/main/validate

No; I see that the script that we call is at `/usr/local/bin/validate.sh` (not under version control, afaik). It is a wrapper to the one that you mention, at the root of the datastore-specifications directory. Probably not the optimal organization ...

**Comment by adf-ncgr** [comment]
OK, thanks. I added that script to the repo and added some of the further details you suggested. Will go ahead with the merge but feel free to revise if anything seems inaccurate/insufficient (I skipped the bit about compressing/indexing since that's more specific to certain types of collections, I think- and probably already covered somewhere under the linked PROTOCOLS I'm guessing).

File: README.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
-All the LIS Datastore YAML files under one roof.
+All the LIS Datastore YAML files (and some others we consider "metadata") under one roof.
 
-This repository contains the description_[Genus]_[species].yml and README.[collection].yml files in the LIS Datastore.
+This repository contains the description_[Genus]_[species].yml and README.[collection].yml files in the LIS Datastore as well as some other files handled as metadata.
+
+Several types of files should be checked for valid format and contents prior to pushing to the datastore-metadata repository:
+
+```
+  validate.sh description_genus_species YAML_FILE
```
Comment (el239) [inline README.md:9]: This script just kept printing the usage at me, I must be doing something wrong. But, I was able to validate the yaml with executable in the parent directory as per its usage example:
./validate readme.schema.json YAML_FILE

File: README.md [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+  validate.sh description_genus         YAML_FILE
+  validate.sh readme                    YAML_FILE
```
Comment (el239) [inline README.md:11]: This script just kept printing the usage at me, I must be doing something wrong. But, I was able to validate the yaml with executable in the parent directory as per its usage example:
./validate readme.schema.json YAML_FILE

Comment (el239) [inline README.md:11]: This script just kept printing the usage at me, I must be doing something wrong. But, I was able to validate the yaml with executable in the parent directory as per its usage example:
./validate readme.schema.json YAML_FILE

Comment (adf-ncgr) [reply README.md:11]: thanks, were you running this on the ISU system or on one of the NCGR LIS servers? I think there's a hard-coded path in that script that may be throwing things off if not run on the former, but it should be pretty easily fixed. I put the script in the repository without thinking about it too much, but TBH I didn't actually know it existed until Steven mentioned it...

Comment (el239) [reply README.md:11]: I was on one of our own, I guess that must be why.

Comment (adf-ncgr) [reply README.md:11]: OK, I think I fixed the hardcoding issue. I think that was indeed causing the behavior you described with it just printing usage. In any case, the alternate (rawer) validation method you figured out should be equivalent, the validate.sh script just does some bit of wrapping of that.

**Pull request merged**